client/my-sites/checkout/checkout/domain-refund-policy.jsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { gaRecordEvent } from 'lib/analytics/ga';
+import { DOMAIN_CANCEL, REFUNDS } from 'lib/url/support';
+import Gridicon from 'components/gridicon';
+import {
+	hasDomainBeingUsedForPlan,
+	hasDomainRegistration,
+	hasDomainRenewal,
+	hasNewDomainRegistration,
+} from 'lib/cart-values/cart-items';
+
+class DomainRefundPolicy extends React.Component {
+	static displayName = 'RegistrationRefundPolicy';
```

Comment:
I guess we can skip this line - the name of the component should be picked up from the default export. And it's confusing to have different names for this component like this anyway.

Reply:
This is a leftover from the old code. Will remove.